src/docs/sphinx/developerGuide/KeyComponents/LvArray.rst

```diff
@@ -39,4 +17,4 @@ The file ``common/DataTypes.hpp`` provides shorthand aliases for commonly used c
 LvArray documentation
 =====================
 
-Please refer to the full `LvArray documentation <https://lvarray.readthedocs.io/en/latest/>`_ for details on each of the classes.
+Please refer to the full :doc:`LvArray documentation <../../../../coreComponents/LvArray/docs/sphinx/index>`
```

Comment:
I'm not sure what the best way to link to the LvArray documentation is. Ideally we'd point them at the version that was just built locally, however when we build the GEOSX docs the doxygen output is in a different place so then the LvArray doxygen links are broken.

Reply:
> Also when linking like this you don't get the doctree side-bar view.

This was one of the reasons I made it just an html link to LvArray's readthedocs. Also attempting to embed it directly into GEOSX's doctree does not produce very good results, since LvArray has its own documentation structure (e.g. different header/section levels).

Reply:
Yeah I'll change it back to a html link. There may be periods where we're behind LvArray develop but it should only be temporary and the docs should still be accurate.